index.html

```diff
@@ -0,0 +1,633 @@
+                    <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 -960 960 960" width="24"><path d="M480-344 240-584l56-56 184 184 184-184 56 56-240 240Z"/></svg>
+                </div>   
+            </div>
+        </div>
+    </div>
+
+    <div class="sidebar">
+        <div class="logo-arrow">
+            <img src="./images/tezo.png" alt="tezo logo" class="logo">  
+            <img src="./images/handle.png" alt="back button" class="backbutton" onclick=hidesidebar()>     
+        </div>     
+        <p class="heading">ALL</p>
+        <div class="dashboard">
+            <div class="dash-left">
+                <img src="./images/bx-lock.png" alt="lock image" srcset="">
+                <span>Dashboard</span>
+            </div>
+            <div class="dash-right">
+                <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 -960 960 960" width="20"><path d="m321-80-71-71 329-329-329-329 71-71 400 400L321-80Z" fill="#808080"/></svg>
+            </div>
+        </div>
+        <div class="dashboard">
+            <div class="dash-left">
+                <img src="./images/bx-lock.png" alt="lock image" srcset="">
+                <span>Roles</span>
+            </div>
+            <div class="dash-right">
+                <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 -960 960 960" width="20"><path d="m321-80-71-71 329-329-329-329 71-71 400 400L321-80Z" fill="#808080"/></svg>
+            </div>
+        </div>
+        <div class="dashboard-seperate">
+            <div class="dash-left">
+                <img src="./images/bx-home-smile.png" alt="smiling house" srcset="">
+                <span class="employee-seperate">Employees</span>
+            </div>
+            <div class="dash-seperate-right">
+                <svg xmlns="http://www.w3.org/2000/svg" height="20" viewBox="0 -960 960 960" width="20"><path d="M480-344 240-584l56-56 184 184 184-184 56 56-240 240Z" fill="#f55252"/></svg>
+            </div>
+        </div>
+        <div class="department-button-left">
+            <div class="greydot"></div>
+            <span>All</span>
+        </div>
+        <div class="department-button-left">
+            <div class="greydot"></div>
+            <span>Active</span>
+        </div>                
+        <div class="department-button-left">
+            <div class="greydot"></div>
+            <span>In Active</span>
+        </div>
+        <p class="heading">DEPARTMENTS</p>
+        
+        <div class="department-button">
+            <div class="department-button-left">
+                <div class="greydot"></div>
+                <span>IT</span>
+            </div>
+            <div class="department-button-square">
+                3
+            </div>
+        </div>
+        <div class="department-button">
+            <div class="department-button-left">
+                <div class="greydot"></div>
+                <span>UI/UX</span>
```

Comment:
span is not a proper tag for links. also all links should have cursor pointer effect on hover.